packages/terra-i18n/src/intlLoaders.js

```diff
@@ -1,29 +1,53 @@
-/* eslint-disable */
+/* eslint-disable import/no-unresolved, compat/compat, no-console */
 import intlLoaders from 'intlLoaders';
 
+const hasIntlData = (locale) => {
```

Comment:
The `intl` documentation shows an[ example of checking if the local data has already been loaded](https://www.npmjs.com/package/intl#intljs-and-node) by using a module called `intl-locales-supported`. That module accepts a list of app locales to determine which locale data should be loaded. This is a flavor of the same tool. [Here is their code for reference](https://github.com/formatjs/formatjs/blob/master/packages/intl-locales-supported/src/index.ts#L7) If we want to eventually worry about internationalized pluralization rules, it might be beneficial to pull it in directly.

Thoughts?

Reply:
https://www.npmjs.com/package/intl-locales-supported

0 dependencies, not much to it. I wouldn't be opposed to pulling it in directly if it suits our needs.

Reply:
+1 let's just pull it in ourselves.